web-client/src/account.rs

```diff
@@ -127,11 +127,88 @@ impl PlainAccount {
     }
 }
 
+#[derive(serde::Serialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+pub struct PlainStaker {
+    // The staker's balance.
+    balance: u64,
+    // The address of the validator for which the staker is delegating its stake for. If it is not
+    // delegating to any validator, this will be set to None.
```

Comment:
```suggestion
    /// The address of the validator for which the staker is delegating its stake for. If it is not
    /// delegating to any validator, this will be set to None.
```